specification/metrics/listcost.md

```diff
@@ -0,0 +1,36 @@
+# List Cost
+
+List Cost represents the cost calculated based on [List Unit Price](#listunitprice) and the corresponding [Pricing Quantity](#pricingquantity). List cost is denominated in the [Billing Currency](#billingcurrency) and is used for calculating savings based on various rate optimization activities, by comparing it with [Effective Cost](#effectivecost).
+
+The ListCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal, denominated in the BillingCurrency, and MUST NOT contain null values. When a ListUnitPrice is not null, multiplying the ListUnitPrice by PricingQuantity MUST produce the ListCost.
+
+In cases where the ListUnitPrice is null, the following applies:
+
+* If the charge is calculated based on other charges (e.g. [ChargeCategory](#chargecategory) is 'Tax'), the ListCost MUST be calculated based on the ListCost of the related charges.
```

Comment:
@udam-f2 Is there guidance around putting conditional statements at the beginning vs end of normative statements?  If not, could I propose that normative statements always start with a MUST, SHOULD, etc, with modifiers to follow?  Something like:

```suggestion
* The ListCost MUST be calculated based on the ListCost of the related charges if the charge is calculated based on other charges (e.g. [ChargeCategory](#chargecategory) is 'Tax').
```

If not, perhaps the opposite, where conditional statements are always up top?  If so, this section is fine, but other specs would need a scrub.

Reply:
performed updates based on suggestion